src/components/header/_macro.spec.js

```diff
@@ -567,110 +392,50 @@ describe('macro: header', () => {
-      expect(values).toEqual(['#1', '#2', '#3']);
-    });
-
-    it('has the provided custom class', () => {
-      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_SINGLE));
-
-      expect($('.ons-header-service-nav--main').hasClass('custom-class')).toBe(true);
-    });
-
-    it('has the provided `aria-label` for the list for mobile', () => {
-      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_SINGLE));
-
-      expect($('.ons-header-service-nav--mobile').attr('aria-label')).toBe('Services menu');
-    });
-
-    it('has the link text for each list item for mobile', () => {
-      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_MULTIPLE));
-
-      const values = mapAll($('.ons-header-service-nav--mobile .ons-header-service-nav__link'), (node) => node.text().trim());
-      expect(values).toEqual(['Title 1', 'Title 2', 'Title 3']);
-    });
-
-    it('has the link href for each list item for mobile', () => {
-      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_MULTIPLE));
-
-      const values = mapAll($('.ons-header-service-nav--mobile .ons-header-service-nav__link'), (node) => node.attr('href'));
-      expect(values).toEqual(['#1', '#2', '#3']);
-    });
-
-    it('renders a button with expected parameters', () => {
       const faker = templateFaker();
       const buttonSpy = faker.spy('button', { suppressOutput: true });
-
       faker.renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_MULTIPLE);
 
-      expect(buttonSpy.occurrences).toContainEqual({
-        text: 'Menu',
-        classes: 'ons-u-d-no ons-js-toggle-services',
-        type: 'button',
-        variants: ['mobile', 'text-link'],
-        attributes: {
-          'aria-label': 'Toggle services menu',
-          'aria-controls': 'service-links',
-          'aria-expanded': 'false',
-        },
+      test('THEN: renders with correct display class on desktop', () => {
+        expect($('.ons-header__links .ons-grid__col').hasClass('ons-u-d-no@xxs@m')).toBe(true);
+      });
+      test('THEN: renders button on mobile', () => {
+        expect(buttonSpy.occurrences).toContainEqual({
+          text: 'Menu',
+          classes: 'ons-u-d-no ons-js-toggle-services',
+          type: 'button',
+          variants: ['mobile', 'text-link'],
+          attributes: {
+            'aria-label': 'Toggle services menu',
+            'aria-controls': 'service-links',
+            'aria-expanded': 'false',
+          },
+        });
       });
     });
-
-    it('renders a button with correct variant if `internal`', () => {
+    describe('WHEN: one item is provided to the itemsList parameter', () => {
+      const $ = cheerio.load(renderComponent('header', EXAMPLE_HEADER_SERVICE_LINKS_SINGLE));
+      test('THEN: renders without multilink display class on desktop', () => {
```

Comment:
```suggestion
      test('THEN: renders without multiple link display class on desktop', () => {
```
Think this wording is better

Reply:
done!